_install-and-configure/install-opensearch/docker.md

```diff
@@ -54,6 +56,14 @@ Before launching OpenSearch you should review some [important system settings]({
    cat /proc/sys/vm/max_map_count
    ```
 
+### Windows settings
+For Windows workloads using WSL through Docker Desktop, run the following commands in terminal to set the `vm.max_map_count`:
```

Comment:
Either "in the terminal" or "in a terminal" (please add article).

Reply:
> Either "in the terminal" or "in a terminal" (please add article).

No article needed because it's a program, but I think capitalizing it should do the job.

Reply:
Isn't it a common noun, as in "command line"?

Reply:
Yes, that would work, too!